apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.ts

```diff
@@ -0,0 +1,8 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-card-link-list-example',
+    templateUrl: 'card-link-list-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
do you need this?

Reply:
removed OnPush ChangeDetectionStrategy. fixed